test_agents.py

```
@@ -0,0 +1,14 @@
+from agents import Agent as OpenAIAgent
+from src.verifact_agents import ClaimDetector
```

Pylint (3.3.7)

[error] No name 'ClaimDetector' in module 'src.verifact_agents'

(E0611)